DataManagementSystem/DB/FileCatalogComponents/FileManagerBase.py

```diff
@@ -1153,10 +1156,11 @@ def getFilesInDirectory( self, dirID, verbose = False, connection = False ):
 
   def getDirectoryReplicas( self, dirID, path, allStatus = False, connection = False ):
     """ Get the replicas for all the Files in the given Directory
-        :param DirID : ID of the directory
-        :param path : useless
-        :param allStatus : whether all replicas and file status are considered
-                          If False, take the visibleFileStatus and visibleReplicaStatus values from the configuration
+
+        :param DirID: ID of the directory
+        :param path: useless
```

Comment:
well, if the thing is not used, it doesn't really matter what type it is. How about `type unused`